linera-client/src/benchmark.rs

```diff
@@ -620,33 +610,41 @@ impl<Env: Environment> Benchmark<Env> {
         Ok(())
     }
 
-    /// Generates information related to one block per chain, up to `num_chains` blocks.
+    /// Generates information related to one block per chain.
     pub fn make_benchmark_block_info(
-        keys: HashMap<ChainId, AccountOwner>,
+        benchmark_chains: Vec<Vec<(ChainId, AccountOwner)>>,
         transactions_per_block: usize,
         fungible_application_id: Option<ApplicationId>,
-    ) -> Vec<(ChainId, Vec<Operation>, AccountOwner)> {
+    ) -> Vec<Vec<(Vec<Operation>, AccountOwner)>> {
```

Comment:
Do we really gain anything by putting together this huge collection, rather than creating the operations directly in the chain (group) tasks, where we need them?